app/controllers/users_controller.rb

```diff
@@ -26,6 +26,8 @@ def edit
   # POST /users.json
```

Comment:
remove these changes - you can do
`git checkout master app/controllers/users_controller.rb`